product_details/__init__.py

```diff
@@ -10,23 +10,27 @@ class MissingJSONData(IOError):
     pass
 
 
-__version__ = '0.14.1'
-__all__ = ['__version__', 'product_details', 'version_compare']
+__version__ = "0.15.0"
```

Comment:
This is a breaking change because of the Python version movement, so let's give it a major bump

```suggestion
__version__ = "1.0.0"
```